lib/kompo.rb

```
@@ -1,452 +1,2030 @@
+    def build_c_extension(dir_name, gem_ext_name, work_dir)
+      ruby = InstallRuby.ruby_path
+      bundle_ruby_dir = BundleInstall.bundle_ruby_dir
+
+      puts "Building C extension: #{gem_ext_name}"
+
+      # Run extconf.rb to generate Makefile (use chdir option for subprocess only)
+      # Unset Bundler environment to prevent interference with system Ruby/Bundler
+      # Set GEM_PATH to bundle standalone gem directory for build-time dependencies (e.g., mini_portile2)
+      # Suppress clang 18+ warning that causes mkmf try_cppflags to fail
+      puts "Running extconf.rb in #{dir_name}"
+      gem_path = bundle_ruby_dir || ""
```

RuboCop (1.82.1)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)